app/controllers/two_factor_authentication/totp_verification_controller.rb

```diff
@@ -4,6 +4,8 @@ class TotpVerificationController < ApplicationController
 
     skip_before_action :handle_two_factor_authentication
 
+    helper_method :confirmation_for_phone_change?
```

Comment:
@monfresh got it, misunderstanding on my part, just updated with that helper removed.